.gitignore

```diff
@@ -19,6 +19,7 @@
 # misc
 .DS_Store
 *.pem
+.idea
```

Comment:
nit: This sounds like it's specific to your personal setup, so placing it in a global `.gitignore` might make more sense (e.g. https://sebastiandedeyne.com/setting-up-a-global-gitignore-file)

Reply:
Unfortunately everybody that uses a webstorm (or any Jetbrains project) has those kind of files :/ 

This is already [present in the portals repo](https://github.com/gigs/portals/blob/3698148f64bf2f88c611d3d1ffed9adc35d74798/.gitignore#L18) but I'm happy to ignore it locally if that's preferable